packages/styles/src/components/badge/badge.stories.js

```diff
@@ -0,0 +1,23 @@
+export default {
+  title: 'Patterns/Badges, Chips, and Tags',
+  argTypes: {
+    variant: {
+      control: { type: 'select' },
+      options: ['', 'danger'],
+      description: 'Adds a modifier class to the component to display the danger variation (see generated HTML)'
+    },
+  },
+};
+
+const BadgeTemplate = (args) => {
+  const badgeType = args.variant ? `cbp-badge--${args.variant}` : null;
+  
+  return `
+    <div class="${badgeType ? 'cbp-badge ' + badgeType : 'cbp-badge'}">
+      5
+    </div>
+  `;
+};
```

Comment:
- Line 12, lets destructure the `args` parameter to show only `variant` (or `danger`)
- Because of the change from the previous comment, we can delete the variable and put a single ternary operator in the class attribute. `cbp-badge` will always be there so only the variant that needs to show/hide is the modifier class (`variant ? 'cbp-badge--danger' : ''`)
- Since we delete the variable, we can replace the brackets with parentheses and remove the return statement since its only returning the markup in the template literal
```
const BadgeTemplate = ({ variant }) => ( `[markup here]`);
```

Reply:
done.

I don't give a lot of thought or priority to optimizing story code. My thinking was to keep the stories more uniform by passing `args` in. It's less re-work if we add more controls later.